### PR TITLE
Improve german courses

### PR DESCRIPTION
**Pull request by relativvv**
## What does this PR do?
This pr improves the courses page for the german courses by a few ones

### Description
This pr improves the courses page for the german courses by a few ones, e.g. SQL and Ansible

### Why is this valuable (or not)?
German people are able to find courses to e.g. sql and ansible easier

### How do we know it's really free?
check the links, everything is free to use

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-de.md [hunk 1]
```diff
@@ -11,9 +13,21 @@
 * [Python](#python)
 * [Rust](#rust)
 * [Spieleentwicklung](#spieleentwicklung)
+* [SQL](#sql)
 * [TypeScript](#typescript)
 
 
+### Ansible
+
+* [Ansible Tutorial](https://www.redhat.com/de/topics/automation/learning-ansible-tutorial) - RedHat Tutorial für Ansible
```
Comment (Thenlie) [inline courses/free-courses-de.md:22]: It looks like this citation should just be `RedHat`
```suggestion
* [Ansible Tutorial](https://www.redhat.com/de/topics/automation/learning-ansible-tutorial) - RedHat
```

Comment (relativvv) [reply courses/free-courses-de.md]: updated 👍

File: courses/free-courses-de.md [hunk 2]
```diff
@@ -77,6 +91,12 @@
 * [Unreal Engine 4 Tutorial Deutsch/German](https://www.youtube.com/playlist?list=PLNmsVeXQZj7olLCliQ05e6hvEOl6sbBgv) - The Morpheus Tutorials
 
 
+### SQL
+
+* [SQL-Grundlagen](https://wiki.selfhtml.org/wiki/Datenbank/SQL-Grundlagen) - SelfHTML
+* [SQL-Island](https://sql-island.informatik.uni-kl.de) - Spielerisch SQL lernen
```
Comment (eshellman) [inline courses/free-courses-de.md:97]: This belongs in a "playgrounds" list

File: courses/free-courses-de.md [hunk 3]
```diff
@@ -11,9 +13,21 @@
...
 
+### Ansible
+
+* [Ansible Tutorial](https://www.redhat.com/de/topics/automation/learning-ansible-tutorial) - RedHat
+
+
+### Bash
+
+* [Bash-Scripting Grundkurs](https://www.ernstlx.com/linux90script.html)
+* [LearnShell](https://www.learnshell.org/de/) - Interaktiv die Shell kennenlernen
```
Comment (eshellman) [inline courses/free-courses-de.md:28]: couldn't navigate this because too many ads

**Review by eshellman — CHANGES_REQUESTED** [review]
remove the two I flagged and your good to go. consider adding the playground in a new PR

File: courses/free-courses-de.md (hunk 2 above)
Comment (Thenlie) [reply courses/free-courses-de.md:97]: @relativvv As an example: https://github.com/EbookFoundation/free-programming-books/blob/main/more/free-programming-playgrounds.md

File: courses/free-courses-de.md (hunk 3 above)
Comment (relativvv) [reply courses/free-courses-de.md]: alright, I removed it

**Comment by eshellman** [comment]
having added the german playground (Thanks!) could you add it to ReadMe.md?

**Comment by relativvv** [comment]
I also added the playground file to the readme

**Comment by eshellman** [comment]
Thanks for sticking with it, and welcome to free-programming-books!

**Pull request merged**